api/store/mongo/migrations/migration_23.go

```diff
@@ -0,0 +1,48 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	migrate "github.com/xakep666/mongo-migrate"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var migration_23 = migrate.Migration{
+	Version: 23,
+	Up: func(db *mongo.Database) error {
+		logrus.Info("Applying migration 23 - Up")
+		mod := mongo.IndexModel{
```

Comment:
`mod` here is too generic. Please use `index_model` or at least `index` so it is clear what it is doing when used below.

You can keep reusing same name for all collections. No problem.